ppa/archive/hathi.py

```diff
@@ -308,10 +308,13 @@ def _content_path(self, ext, ptree_client=None):
         # - don't rely on them being returned in the same order on every machine
         parts = pairtree_obj.list_parts(self.content_dir)
         # find the first zipfile in the list (should only be one)
-        filepath = [part for part in parts if part.endswith(ext)][0]
-
-        return os.path.join(pairtree_obj.id_to_dirpath(), self.content_dir, filepath)
-
+        filepaths = [part for part in parts if part.endswith(ext)]
+        return (
```

Comment:
I think we need some kind of output or logging error/warning if this happens, because if the expected file is not found it means something is wrong with the pairtree data — would be good to know which record it's happening on so we can investigate

Reply:
I would prefer to log it and not raise an exception because there appear to be a variety of reasons that pairtrees can break down, and the cost of raising an exception is high.

Reply:
Was thinking an exception here that we can catch in the indexing script so we can control reporting. Possibly fewer part of the code that need to be touched.

Logging should be ok as long as it's visible when we run the script

Reply:
I see, sorry I didn't see your 2nd comment. yes as long as we're ok with catching it at index function level. I'll do that now